src/farcasterd/trade_state_machine.rs

```diff
@@ -1171,6 +1193,22 @@ fn attempt_transition_from_restoring_swapd_to_swapd_running(
             runtime.handle_failed_connection(event.endpoints, source.clone())?;
             expect_connection = false;
         }
+        (BusMsg::Ctl(CtlMsg::Hello), source) if trade_role == TradeRole::Maker => {
+            if let Some(node_id) = expected_counterparty_node_id {
+                if source.node_addr()
+                    == Some(NodeAddr::new(
+                        node_id,
+                        runtime.config.get_bind_addr().unwrap(),
+                    ))
+                {
+                    info!(
+                        "{} | Peerd connection for restored swap",
```

Comment:
Shouldn't this be the same as `"{} | Peerd connected for restored swap"`?

Reply:
1908a76b08e8dfcb6283e921fb51fcad88c758d7